chart/inlets-operator/README.md

```diff
@@ -73,3 +73,4 @@ Parameter | Description | Default
 `nodeSelector`          | Node labels for data pod assignment                                             | `{}`
 `tolerations`           | Node tolerations                                                                | `[]`
 `affinity`              | Node affinity policy                                                            | `{}`
+`secretKeyRequired`     | If we are using a provider that requires a secret key as well as an access key  | `false`
```

Comment:
Confused about this Boolean? Can you walk me through it?

Reply:
Some providers require a secret access key (ec2 and scaleway) this adds the volume mounts to the operator and passes the flag to the client

Reply:
Can this be done by accepting a secret access key instead?

Reply:
It *could*, i was just trying to keep the secret out of the command, and out of the manifest,  same as the access key that other providers use.

Reply:
What do you think to replicating the same UX as `accessKeyFile` and then checking for whether a value is set to determine this behaviour

Reply:
the UX on that flag is wrong IMO, its not a variable, and if you changed `accessKeyFile` in the values.yaml the installation wouldn't work. 

The chart depends on the secret being created for these keys, and all we need to know is if a secretAccessKey secret is needed. The value of `accessKeyFile` should always be `/var/secrets/inlets/inlets-access-key` 

I could change the "bool" to a string, and say "it should be set to `/var/serets/inlets/secret-access-key` and the just check if the string length is > 0 ?

Reply:
I am just very confused by "secretKeyRequired" as a flag to a helm chart, it seems like something we should be ascertaining from inputs, not having as an input

Reply:
there is no input, the secret is created outside the chart, and we need some indication that they are using a provider that requires both. Hard coding to a set of providers that we pick up is possible, but would require more maintenance. I have changed it to a string, and any non-null value will implement